### ISSUE TITLE
Projects Not Displaying On The Page Chosen In Settings

### ISSUE THREAD

**Issue by stuartduff** [woothemes/projects#102]
The page chosen in the settings for projects no longer seems to be displaying an archive of Projects in any theme. Several customers have reported the same issue with v1.4.1 and I can replicate this on my support installs on themes like Canvas, Twenty Twelve ( Screenshot Below ) etc.

Screenshot: http://cld.wthms.co/1h1Pb

**Comment by jeffikus** [comment]
@stuartduff what are your permalink settings please?

**Comment by stuartduff** [comment]
@jeffikus here ya go man http://cld.wthms.co/1g4xs :)

**Comment by jeffikus** [comment]
Thanks, do me a solid and change it to Day and Name, the second option, and let me know if that makes a difference?

**Comment by stuartduff** [comment]
@jeffikus Update on this one and it gets a little stranger as I got to dig a little deeper for you today.

Okay firstly Projects functions on Twenty Twelve without WooCommerce activated and the permalinks set to any configuration, default, postname etc, etc.

Screenshot: http://cld.wthms.co/1aGy7

Now If I enable WooCommerce the projects stop displaying and will only display again if the default permalink structure is chosen for posts and saved.

Screenshot: http://cld.wthms.co/15TvO

So with projects and WooCommerce enabled only these permalink settings allow projects to display.

Screenshot: http://cld.wthms.co/rxtj

Let me know if this additional info helps

**Comment by iulianmarius** [comment]
I still have this issue. Projects are shown only with default permalink settings. Do you have an update on this?

**Comment by pmciano** [comment]
I have a [user with a similar issue](https://woothemes.zendesk.com/agent/tickets/352679).

On a local site using *Canvas 5.9.12* with *WooCommerce 2.5.0-dev* activated and permalinks set to **Day and name**, I activated the Projects plugin. I created a Projects page, set it as the plugin's **Projects Page**, and created my first project.

The project was not viewable on its own, single project page, nor was it visible on the Projects page I set for the plugin.

As per [stuartduff's comment](https://github.com/woothemes/projects/issues/102#issuecomment-73918355), I tried deactivating WooCommerce and switching to the Twenty Twelve theme. This did not address the issue.

However, I toggled the permalinks to the **Default** setting, then back to **Day and name**, and that did the trick. Even after I switched back to Canvas and re-activated WooCommerce, both pages were still working.

I hope that helps.
